### PR TITLE
Hide OpenLMIS settings behind a toggle

### PR DESCRIPTION
**Pull request by kaapstorm**
For context see [FB 170111](http://manage.dimagi.com/default.asp?170111).

Labelling "hold off on merge" because in local dev environment I can't seem to get the CommTrack / CommCare Supply settings sidebar to appear, so I can't confirm that this change addresses the issue. I must be missing something small. (Yes, CommTrack is enabled for the domain.) Will remove label once I can see it working, but pushing in case whatever I'm missing is obvious to you.

@czue, is this what you had in mind re the toggle? Anything else you think I should put behind it (maybe [the OpenLMIS task](https://github.com/dimagi/commcare-hq/blob/master/custom/openlmis/tasks.py#L15))? I checked Confluence, and it has a whole section on OpenLMIS, but not how to enabled it, so it seems there's nothing that needs to be changed there -- unless I missed something.

Buddy @snopoke

### CODE REVIEW

**Comment by czue** [comment]
did you try enabling the commtrack feature preview?

File: corehq/apps/hqwebapp/models.py [hunk 1]
```diff
@@ -544,17 +545,21 @@ def sidebar_items(self):
                     'title': CommTrackSettingsView.page_title,
                     'url': reverse(CommTrackSettingsView.urlname, args=[self.domain]),
                 },
-                # external sync
-                {
-                    'title': FacilitySyncView.page_title,
-                    'url': reverse(FacilitySyncView.urlname, args=[self.domain]),
-                },
+                # (external sync goes here, if OpenLMIS toggle is enabled)
```
Comment (czue) [inline corehq/apps/hqwebapp/models.py]: i think it'd be fine to stick it at the end, but not blocking

**Comment by czue** [comment]
this looks good to me and I think all that's needed for now. agree would be good for you to be able to test locally before merge though. let me know if i can help with that

**Comment by kaapstorm** [comment]
Found the problem with the sidebar: I was passing a `namespace` param to `enabled()`, and it was raising a TypeError that I missed. I can confirm the sidebar now works as expected.

**Pull request merged**